src/store/sparql.py

```diff
@@ -1,13 +1,21 @@
+import os
+
+from typing import Dict
+from SPARQLWrapper import SPARQLWrapper, QueryResult, JSON
 
 from store.base import BaseStore
-import os
-from SPARQLWrapper import SPARQLWrapper, JSON, QueryResult
+
+def get_value_from_dict(item, name: str):
+    """this function will aquire a value from a given list of dictionaries"""
+    
+    return item[name]["value"]
 
 class SparqlStore(BaseStore):
 
     #seting up the self.url
     def setup(self):
         url = os.environ.get("SPARQL_ENDPOINT_URL", "https://beta.gss-data.org.uk/sparql")
+        self.host = os.environ.get("HOST", "local host and the port it is running on")
```

Comment:
"local host and the port it is running on" should literally just be "http:localhost:5000". That way when we run this without the HOST env var being set (i.e we run it locally) the urls will resolve correctly.